### ISSUE TITLE
Latest Posts Block - post title is not accessible when full post content is included

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#15376]
When working on the updates to the Latest Posts Block I noticed that when the articles are displayed along with post content the markup isn't particularly accessible.

<img width="755" alt="Screen Shot 2019-04-30 at 13 54 18" src="https://user-images.githubusercontent.com/444434/56962928-96e9bd80-6b4f-11e9-8e8f-9f240645a6bd.png">

In this context my questions are:

1. Should the article be wrapped in an `<article>` tag as it is now a full piece of content and not just a title or title + excerpt. If feels like [syndicatable content in its own right](https://www.w3.org/TR/html5/semantics.html#the-article-element).
2. Given the above, should the title of the post be wrapped in a valid heading level tag? You will see from the screenshot above that if the article content contains headings (which it likely will) then the markup of the Block becomes quite confusing. It _might_ be less so if it were within an article.

_Originally posted by @getdave in https://github.com/WordPress/gutenberg/pull/14627#issuecomment-487942052_

**Comment by afercia** [comment]
Interesting issue @getdave. To my understanding, this happens only when the block setting "Show full post" is on.

When this happens, I'd be more worried for the headings hierarchy and general semantics of the content which becomes... messy? 🙂 The headings hierarchy can be disrupted by the headings within the latests posts content. The latests posts can basically contain anything, I'm afraid there's really no way to guarantee correct semantics. Wondering if the setting "Show full post" is a good idea in the first place.

That said, I may be wrong, but I don't know of any assistive technology that communicates the semantics of `<article>`.

**Comment by ramonjd** [comment]
If folks still think this issue is relevant in 2025, please reopen. Thank you!
